features/src/main/feature/camel-features.xml

```diff
@@ -107,6 +107,17 @@
         <bundle dependency="true">wrap:mvn:org.apache.httpcomponents.client5/httpclient5/5.2.1</bundle>
     </feature>
 
+    <feature name="netty" version="4.1.106.Final">
+        <bundle dependency='true'>mvn:io.netty/netty-buffer/4.1.106.Final</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-common/4.1.106.Final</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-codec/4.1.106.Final</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-handler/4.1.106.Final</bundle>
+        <bundle dependency='true'>mvn:io.netty/netty-handler-proxy/4.1.106.Final</bundle>
```

Comment:
`netty-handler-proxy` should be removed from the netty feature because it is not so commonly used

Reply:
Ok, done